learning_resources_search/models.py

```diff
@@ -37,6 +43,8 @@ def source_label(self):
 
     def source_description(self):
         channel = self.source_channel()
```

Comment:
Outside of this scope's PR changes, but should this function also take the `source_type` into consideration?

```python
    def source_channel(self):
        original_query_params = self.original_url_params()
        channels_filtered = Channel.objects.filter(search_filter=original_query_params)
        if channels_filtered.exists() and self.source_type ==  CHANNEL_SUBSCRIPTION_TYPE:
            return channels_filtered.first()
        return None
```

Reply:
potentially. - im currently working on another PR that involves sending actual emails for percolated searches so this might change there